src/res/translation/translation_de_DE.ts

```diff
@@ -1085,12 +1085,12 @@ Wir haben Deinen Kanal stummgeschaltet und die Funktion &apos;Stummschalten&apos
     <message>
         <location filename="../../clientdlg.cpp" line="207"/>
         <source>If this LED indicator turns red, you will not have much fun using the %1 software.</source>
-        <translation type="unfinished"></translation>
+        <translation>Wenn diese LED rot leuchtet, dann wirst du keinen Spaß mit der %1 Software haben.</translation>
```

Comment:
```suggestion
        <translation>Wenn diese LED rot leuchtet, dann wirst du wenig Spaß mit der %1 Software haben.</translation>
```
... would sound less drastic?

"mit der Jamulus Software" still sounds strange to me, but matches what's there in English, so I guess we'd keep it?
Personally, I'd just say "mit Jamulus haben".

Reply:
Exactly. I think we should remove the "Software" part one time. Probably it's due to historical reasons (llcon software)?

Reply:
I've opened #2350 to track this.